backend/alembic/versions/0002_create_skills.py

```diff
@@ -17,12 +17,10 @@
 UPGRADE_SQL = """
 CREATE TABLE IF NOT EXISTS `skill` (
     `Skill_ID` int NOT NULL AUTO_INCREMENT,
-    `Course_ID` varchar(20) NOT NULL,
     `Skill_Name` varchar(50) NOT NULL,
     `Skill_Desc` varchar(255),
     `Is_Active` boolean,
     PRIMARY KEY (`Skill_ID`),
-    FOREIGN KEY `FK1_role_skill` (`Course_ID`) REFERENCES course(`Course_ID`)
```

Comment:
Can u create a new alembic version to reflect this change? Alembic versions shouldnt be changed after merge